src/test/kotlin/uk/gov/justice/hmpps/offendersearch/services/PhraseResultParserKtTest.kt

```diff
@@ -1,15 +1,21 @@
 package uk.gov.justice.hmpps.offendersearch.services
 
-import com.google.gson.Gson
+import com.fasterxml.jackson.databind.ObjectMapper
 import com.nhaarman.mockito_kotlin.mock
 import com.nhaarman.mockito_kotlin.whenever
 import org.assertj.core.api.Assertions.assertThat
 import org.elasticsearch.search.SearchHit
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.json.JsonTest
 import uk.gov.justice.hmpps.offendersearch.dto.OffenderDetail
 
+@JsonTest
 internal class PhraseResultParserKtTest {
+  @Autowired
+  lateinit var objectMapper: ObjectMapper
```

Comment:
had to switch to jackson here - seemed easier than leaving as gson and then adding the localdate converters...